modules/nvidia_plugin/src/CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ target_include_directories(${OBJ_NAME}
 )
 target_include_directories(${OBJ_NAME}
                            PUBLIC
+                           "${CUDNN_INCLUDE_DIR}"
```

Comment:
@ilya-lavrenov could you take a look? Without it cudnn.h wasn't found even if I specified `INTERFACE_INCLUDE_DIRECTORIES "${CUDNN_INCLUDE_DIR}"`

Reply:
It's strange because we have linkage with `CUDA::cutensor` below and it should populate headers we set before.
But let's avoid spending extra time on it :)